### ISSUE TITLE
Integrate updater into server code

### ISSUE THREAD

**Issue by geek1011** [geek1011/BookBrowser#19]
It would be better to have to updater check on the server side.

**Comment by SR-G** [comment]
I agree - this would also allow to have a "lazy start", starting immediately the server (exposing the UI immediately) and having the initial indexing done in the background, with results populated on the fly. Would then probably need a small status in the UI about that in order to display "indexing in progress XX/YY".

**Comment by geek1011** [comment]
@SR-G sorry, but you interpreted this issue wrong. By updater, I mean the part (currently disabled), which checks for a new BookBrowser release. 

As for lazy start, feel free to open an issue. I've actually been thinking about that for a bit.

**Comment by SR-G** [comment]
You are right of course, sorry, have really thought that you were willing to integrate a "book repository update" inside the UI (and not only on start). Going to open an additional issue.

**Comment by geek1011** [comment]
@SR-G it's ok
